src/App.js

```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Great job so far :clap However According to the project requirements The App.js component should also be refactored into class-based. 

![18](https://user-images.githubusercontent.com/79058364/197874097-23ab1ad1-ac9a-431d-bbdd-8daae03ca565.png)


```ruby
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

Reply:
Sorry, I understood all components in the folder components, I did not think that I should change the main App, I will work on it